internal/bminventory/inventory.go

```diff
@@ -4184,3 +4184,21 @@ func (b *bareMetalInventory) configureStaticIPs(cluster *common.Cluster, staticI
 
 	return strings.Join(lines, "\n")
 }
+
+func (b *bareMetalInventory) DownloadBootFiles(ctx context.Context, params installer.DownloadBootFilesParams) middleware.Responder {
```

Comment:
tests? 
And maybe it should be a different handler, it's not related to any cluster/host components.

Reply:
Moved to new handler and added tests, 100% coverage